### PR TITLE
EPMDJ-4369: Notifications - Bulk Operations integration

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: 3415

### CODE REVIEW

File: src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx [hunk 1]
```diff
@@ -6,9 +6,10 @@ import {
 } from '@drill4j/ui-kit';
 
 import { Notification as NotificationType } from 'types/notificaiton';
-import { Notification } from './notification/notification';
+import { Notification } from './notification';
 
 import styles from './notifications-sidebar.module.scss';
+import { deleteAllNotifications, readAllNotifications } from './api';
```
Comment (RomanAchilov) [inline src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx:12]: 👎 up

File: src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx [hunk 2]
```diff
@@ -38,8 +39,16 @@ export const NotificationsSidebar = notificationsSidebar(
           {notifications.length > 0 ? (
             <Content>
               <ActionsPanel align="end">
-                <span onClick={() => {}} data-test="notification-sidebar:mark-all-as-read">Mark all as read</span>
-                <span onClick={() => {}} data-test="notification-sidebar:clear-all">Clear all</span>
+                <span
+                  onClick={() => readAllNotifications({ onError: setErrorMessage })}
+                  data-test="notification-sidebar:mark-all-as-read"
+                >Mark all as read
```
Comment (RomanAchilov) [inline src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx:45]: 👎

File: src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx [hunk 3]
```diff
@@ -38,8 +39,16 @@ export const NotificationsSidebar = notificationsSidebar(
...
+                </span>
+                <span
+                  onClick={() => deleteAllNotifications({ onError: setErrorMessage })}
+                  data-test="notification-sidebar:clear-all"
+                >Clear all
```
Comment (RomanAchilov) [inline src/components/toolbar/notifications-sidebar/notifications-sidebar.tsx:50]: 👎

**Review by RomanAchilov — CHANGES_REQUESTED** [review]
pls pay more attention

**Review by RomanAchilov — APPROVED** [review]

**Review by pavyarov — APPROVED** [review]

**Pull request merged**